### DISCUSSION TITLE
[Question][Hw4] Can we modify the parameter of `mutate` function?

### DISCUSSION THREAD

**Discussion by seyoungsong** [prosyslab-classroom/cs348-information-security#580] [Q&A]
Can we modify the parameter of `mutate` function? I need `seeds` to implement it.
If we can't, I can modify env to have seeds in it, or modify `run` to do some things that `mutate` should do.
But I think this is not a good solution.

TA said
> please do not change the parameter of functions.
https://github.com/prosyslab-classroom/cs448-2021-spring/issues/114#issuecomment-843866780

**Comment by Junseon00** [comment]
well... I think you can modify the `mutate`, if you want, as in the `README.md` , but `run` should not be fixed, except the `NOT implemeted` part.
